week2/homework/src/todoList

```diff
@@ -0,0 +1 @@
+
```

Comment:
This file and `todopromise.txt` shouldn't be committed because they are user-specific and might contain sensitive data. They should be ignored using `.gitignore`.